README.md

```diff
@@ -78,9 +79,22 @@ As fluent-plugin-datadog is an output_buffer, you can set all output_buffer prop
 
 ### Docker and Kubernetes tags
 
-Whether you are using kubernetes, you can enrich your logs with docker and kubernetes tags using [fluent-plugin-kubernetes_metadata_filter](https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter).
-Add the following code to your configuration file to enable the filter plugin:
-```xml
+Tags in Datadog are critical to be able to jump from one part of the product to the other. Having the right metadata associated to your logs is therefore important to jump from the container view or any container metrics to the most related logs.
+
+If your logs contain any of the following attributes, it will automatically be added as Datadog tags on your logs:
+
+* kubernetes.container_image
```

Comment:
I think it's worth mentioning that all tags will be renamed and provide the list of the new names.

Reply:
@ajacquemot updated